ci/deploy.sh

```diff
@@ -138,7 +138,11 @@ Packager: Wez Furlong <wez@wezfurlong.org>
 License: MIT
 URL: https://wezfurlong.org/wezterm/
 Summary: Wez's Terminal Emulator.
+%if 0%{?suse_version}
+Requires: dbus-1, fontconfig, openssl, libxcb1, libxkbcommon0, libxkbcommon-x11-0, libwayland-client0, libwayland-egl1, libwayland-cursor0, Mesa-libEGL1, libxcb-keysyms1, xcb-util-wm-devel
```

Comment:
That does indeed look suspect and should not be there. I think I was trying to find packages that provide similar functionality but made a mistake with that one (at least). Trouble is xcb-util-wm by itself does not exist on opensuse. On Fedora it seems to provide     libxcb-ewmh.so.2()(64bit) and libxcb-icccm.so.4()(64bit) which seem to be standalone packages themselves. They are already installed on my test systems.

After some testing I realize what happened. Those two packages are pulled when that -devel package is installed. That must be why it worked - and perhaps why I picked it in the first place. (Search 'what provides these' on opensuse packages sites etc).

I will replace the devel package with specific packages then test the built RPM to see if it works.

Reply:
I was able to test the tumbleweed RPM which indeed works fine. I think the leap build failed because the repo changed from under it while building. Can we manually run the workflow again?

Reply:
I just did a quick check of the leap RPM. It looks good.